tests/conftest.py

```diff
@@ -31,8 +45,8 @@ async def fresh_pool(event_loop):
         An aiomysql pool connected to the db
     """
     pool = await aiomysql.create_pool(
-        host=os.getenv("CONTAINER_NAME_DB"),
-        port=int(os.getenv("DB_PORT_CONTAINER")),
+        host="localhost",
+        port=int(os.getenv("DB_PORT_HOST")),
```

Comment:
We've used `os.getenv` here but `os.environ[]` elsewhere - is this significant?

Reply:
I think this one should probably get environ really, cos we want to error if it is not set

Reply:
```suggestion
        port=int(os.environ["DB_PORT_HOST"]),
```